### ISSUE TITLE
neubot-server should be python 3 compatible

### ISSUE THREAD

**Issue by DavideAllavena** [bassosimone/neubot-server#47]
We need to run neubot-server even with python3

**Comment by bassosimone** [comment]
I agree. Let's do the easier cleanups first, then we can concentrate on this.

**Comment by bassosimone** [comment]
Changed to a later milestone, since we know that the server runs Python 2 and that is not going to change soon. Better to focus on other objectives, then.
